src/wakelock.c

```diff
@@ -45,10 +44,12 @@ void belle_sip_wake_lock_init(JNIEnv *env, jobject pm) {
 	} else {
 		belle_sip_warning("bellesip_wake_lock_init(): the wakelock system has already been initialized");
 	}
+	if (ctx.powerManager == NULL) {
```

Comment:
Since we don't set `ctx.jvm` to NULL anymore, we need to check outside of the `if(ctx.jvm == NULL)` block, to assign `ctx.powerManager`